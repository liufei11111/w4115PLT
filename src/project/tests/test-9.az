Int a;
Int b;
Int c;
Int main(Int argc, String argv){
b = 2;
a = 10;
if (a > b){
	c = 1;
}
else
	c = 0;

if (a > b) 
	c = 1;
else 
	c =0;

if (a > b){
	c =1;
}

if (a > b)
	c = 1;

if( a > b)
	c = 1;
else {
	c = 0;
}
if ( a >= b )
	c = 1;
if ( a < b)
	c = 1;
if ( a <= b)
	c = 1;
if ( a == b)
	c = 1;
if (a != b)
	c = 1;

Boolean b1;
Boolean b2;
b1 = true;
b2 = false;
if (b1)
	c = 1;
if (b1 && b2)
	c = 1;
if (b1 || b2)
	c = 1;
}