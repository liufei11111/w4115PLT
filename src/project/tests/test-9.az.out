stmts:

int a;
int b;
int c;
b = 2;
a = 10;
if (a > b) {
  c = 1;
}
else c = 0;
if (a > b) c = 1;
else c = 0;
if (a > b) {
  c = 1;
}
else {
  }
if (a > b) c = 1;
else {
  }
if (a > b) c = 1;
else {
  c = 0;
}
if (a >= b) c = 1;
else {
  }
if (a < b) c = 1;
else {
  }
if (a <= b) c = 1;
else {
  }
if (a == b) c = 1;
else {
  }
if (a != b) c = 1;
else {
  }
bool b1;
bool b2;
if (b1) c = 1;
else {
  }
if (b1 && b2) c = 1;
else {
  }
if (b1 || b2) c = 1;
else {
  }

fdecls:



